frontend/src/components/dashboard/MineSearch.js

```diff
@@ -82,6 +83,7 @@ export class MineSearch extends Component {
     } else {
       return (
         <Input
+          defaultValue={this.props.searchValue ? this.props.searchValue : null}
```

Comment:
is undefined preferred? Made the switch.

Reply:
The reason I asked is because I assumed `this.props.searchValue ?` is checking if the value is present or `undefined`. In which case, the entire ternary is unnecessary.

Looking at it now, I realize that it could be just checking if the string is empty or not. In that case, we probably don't need to change anything here at the moment. But it could be the impetus to discuss our use of bottom values (`null` and `undefined`, making sure we're consistent).